contents/tutorials/how-to-rename-events.mdx

```diff
@@ -0,0 +1,67 @@
+---
+title: How to rename events
+sidebar: Docs
+showTitle: true
+author:
+  - bijan-boustani
+date: 2024-10-22
+tags:
+  - actions
+  - events
+---
+
+**Estimated reading time:** 3 minutes ☕☕
+
+**tl;dr** Use [actions](/docs/data/actions). Start capturing events with the new name, and then create an action to group it with the existing event.
+
+## Managing events
+
+After you've started capturing events, you can find them in the [data management](https://us.posthog.com/data-management/actions) tab in the left navigation.
+
+For example, you might have an existing event named `register_button_clicked` and you want to rename it to `signup_button_clicked`.
+
+But once an event is logged, there is no option to rename it!
+
+## How to create an action
+
+The solution is to start sending events with the new name, and then create an action. Actions are a way of combining several related events into one. You can match both the old and new event name and get a fluent transition in your insights, without losing historical data.
+
+Start by clicking the "New action" button in the [actions](https://us.posthog.com/data-management/actions) tab in data management. Then, select the "from event or pageview" option.
+
+<ProductScreenshot
+    imageLight="https://res.cloudinary.com/dmukukwp6/image/upload/create_first_action_light_63d4f8aa42.png"
+    imageDark="https://res.cloudinary.com/dmukukwp6/image/upload/create_first_action_dark_bb2ec7a894.png"
+    alt="Create an action"
+    classes="rounded"
+/>
```

Comment:
We also sometimes use https://screen.studio/ to record demos for more complicated flows. This is probably too simple for it though